### ISSUE TITLE
Symbol level bounding box information in hOCR output

### ISSUE THREAD

**Issue by arvidj** [tesseract-ocr/tesseract#135]
Hello, I'm working on adding symbol level bounding box information in hOCR output, in order to be able to find the position of individual symbols inside words. Is this something that you would like to integrate? I've only had a cursory glance on the hOCR reference, but it doesn't seem to be violate anything. Output would look like:

```
    <span class="ocrx_word" dir="ltr" id="word_1_26" lang="eng" title=
    "bbox 102 194 177 224; x_wconf 90" xmlns=
    "http://www.w3.org/1999/xhtml"><strong><span class="ocrx_symbol" id=
    "symbol_1_26_81" title=
    "bbox 102 200 117 224; x_wconf 92">q</span><span class="ocrx_symbol" id=
    "symbol_1_26_82" title=
    "bbox 121 200 135 218; x_wconf 90">u</span><span class="ocrx_symbol" id=
    "symbol_1_26_83" title=
    "bbox 140 194 143 218; x_wconf 97">i</span><span class="ocrx_symbol" id=
    "symbol_1_26_84" title=
    "bbox 146 200 161 218; x_wconf 92">c</span><span class="ocrx_symbol" id=
    "symbol_1_26_85" title=
    "bbox 163 194 177 218; x_wconf 96">k</span></strong></span>
```

**Comment by jbreiden** [comment]
Speaking only for myself. It's soooooo expensive. That's why I don't bother in PDF output.

**Comment by tfmorris** [comment]
If this were to be contemplated, it should be controlled by a config variable with defaults to false, similar to `hocr_font_info`.  I'd be curious about the use case though. If hOCR is being used as a format to connect two programs, it seems like a very bulky & low fidelity channel. Wouldn't just calling the Tess result iterator yourself give you better control at less cost?

**Comment by amitdo** [comment]
I don't think it is a good idea to add this to the hocr output. A better alternative is to add new simple text renderer that outputs only info about symbols including 'alt choices' and confidence.

**Comment by amitdo** [comment]
The feature was implemented in #310
